tests/test_TFUtil.py

```diff
@@ -143,6 +143,26 @@ def test_Data_unknown_feature_no_time():
   assert d1.batch_shape == (None, None)
 
 
+def test_Data_copy_with_feature_dim_axis():
+  import numpy as np
+  size_placeholder = tf.constant(np.full((10,), 10), dtype=tf.int32)
+  d = Data(name='test_data', shape=(None, 64, 64), dtype='float32',
+           size_placeholder={0: size_placeholder}, batch_dim_axis=0,
+           time_dim_axis=1, feature_dim_axis=3)
+  d_copy = d.copy_with_feature_dim_axis(1)
+  assert d_copy.shape == (64, None, 64)
+  assert d_copy.batch_dim_axis == 0
+  assert d_copy.time_dim_axis == 2
+  assert d_copy.feature_dim_axis == 1
+  assert d_copy.size_placeholder.keys() == [1]
```

Comment:
The check is incorrect I think. Should be:
```suggestion
  assert list(d_copy.size_placeholder.keys()) == [1]
```